concordia/documents.py

```diff
@@ -143,3 +146,44 @@ def get_queryset(self):
                 "asset__item", "asset__item__project", "asset__item__project__campaign"
             )
         )
+
+
+@asset.doc_type
+class AssetDocument(DocType):
+    item = fields.ObjectField(
+        properties={
```

Comment:
Should we include the item creation/modification time?

Reply:
We don't have `created_on` or `updated_on` for items either.